lib/src/base-styles/_nx-colors.scss

```diff
@@ -95,6 +115,9 @@
     // NxGlobalHeader
     --nx-color-global-header-border: var(--nx-color-site-background);
 
+    //Input variables
+    --nx-color-input-text: var(--nx-swatch-indigo-70);
```

Comment:
> Might need to confirm with @JasonDSwearingen regarding placeholder text and the input text. Looks like Kaleido doesn't have any information about the two states?

The mocks don't, but the placeholder text is essentially the disabled text, so we discussed what the colour should be in dark mode, which was indigo-40 in the end.

Reply:
> > Might need to confirm with @JasonDSwearingen regarding placeholder text and the input text. Looks like Kaleido doesn't have any information about the two states?
> 
> The mocks don't, but the placeholder text is essentially the disabled text, so we discussed what the colour should be in dark mode, which was indigo-40 in the end. The input text needed to be specified since the dark mode colour is off from `nx-color-text` in the mocks.

As mentioned above, checked with Jason and you're right. Updated here [56c14de](https://github.com/sonatype/sonatype-react-shared-components/pull/1034/commits/56c14dedd578693242a35993e2ab9db90c9df444)